### PR TITLE
test: added hide-show token coverage. Improved @sendft steps. Combined "open dialog window" steps

### PR DESCRIPTION
**Pull request by NeaRbYjkee**

### CODE REVIEW

File: apps/nfid-frontend-e2e/src/steps/when.ts [hunk 1]
```diff
@@ -107,3 +107,43 @@ When(/^User clicks the back button in Send window$/, async () => {
     Assets.switchSendType,
   )
 })
+
+When(/^User clicks the (.*) option button$/, async (option: string) => {
+  await Assets.getTokenOption(option).click()
+  await browser.pause(10000)
```
Comment (NeaRbYjkee) [inline apps/nfid-frontend-e2e/src/steps/when.ts:113]: this pause is needed as hide/show process takes 8-10 sec and there is no any element to use waitUntil

File: apps/nfid-frontend-e2e/src/steps/when.ts [hunk 2]
```diff
@@ -107,3 +107,43 @@ When(/^User clicks the back button in Send window$/, async () => {
...
+})
+
+When(/^User clicks the ShowHide button of (.*) token$/, async (tokenName: string) => {
+  await Assets.ManageTokensDialog.tokenShowHideButton(tokenName).then(async (it) => {
+    await it.waitForDisplayed()
+    await browser.pause(1500)
+    await it.click()
+    await browser.pause(10000)
```
Comment (NeaRbYjkee) [inline apps/nfid-frontend-e2e/src/steps/when.ts:121]: this pause is needed as hide/show process takes 8-10 sec and there is no any element to use waitUntil

File: apps/nfid-frontend-e2e/src/pages/page.ts [hunk 3]
```diff
@@ -57,6 +57,21 @@ export class Page {
...
   }
 
+  public async clickOnLeftUpperCorner(){
+    await browser.performActions([
+      {
+        type: 'pointer',
+        id: 'mouse',
+        parameters: { pointerType: 'mouse' },
+        actions: [
+          { type: 'pointerMove', x: 0, y: 0 }, // Перемещение указателя в верхний левый угол
```
Comment (olekid) [inline apps/nfid-frontend-e2e/src/pages/page.ts:67]: rus -> eng comments pls

File: apps/nfid-frontend-e2e/src/steps/when.ts [hunk 4]
```diff
@@ -107,3 +107,43 @@ When(/^User clicks the back button in Send window$/, async () => {
...
+})
+
+When(/^User filters tokens by (.*)$/, async (token: string) => {
+  await Assets.ManageTokensDialog.filterField().setValue(token)
+})
+
+When(
+  /^User sets the token ([^"]+) to be displayed if needed$/,
+  async (tokenName: string) => {
+    console.log(await Assets.tokenLabel(tokenName).isDisplayed())
```
Comment (olekid) [inline apps/nfid-frontend-e2e/src/steps/when.ts:132]: console.log

File: packages/ui/src/atoms/dropdown/option.tsx [hunk 5]
```diff
@@ -23,6 +23,7 @@ export const DropdownOption = ({
 }: IDropdownOption) => {
   return (
     <div
+      id={`option`}
```
Comment (olekid) [inline packages/ui/src/atoms/dropdown/option.tsx:26]: now all options will have id = "option", id should be unique for every dom element, so you need to pass custom id to options, where they are used

Comment (olekid) [reply packages/ui/src/atoms/dropdown/option.tsx:26]: @vitaliipustovyi ptal

File: apps/nfid-frontend-e2e/src/steps/when.ts (hunk 1 above)
Comment (olsemeno) [reply apps/nfid-frontend-e2e/src/steps/when.ts:113]: it have to be immidiately

**Review by olsemeno — APPROVED** [review]

**Pull request merged**